### PR TITLE
chore(test): add initial unit tests

### PR DESCRIPTION
**Pull request by devCrossNet**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/universal/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What modules are related to this pull-request**
- [ ] express-engine
- [ ] grunt-prerender
- [ ] gulp-prerender
- [ ] hapi-engine
- [ ] universal-next
- [x] universal
- [ ] webpack-prerender

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
adding possibility to run jasmine tests and some unit tests for platform-node

* **What is the current behavior?** (You can also link to an open issue here)
no tests

* **What is the new behavior (if this is a feature change)?**
some unit tests

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: helpers/polyfills.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require('zone.js');
```
Comment (gdi2290) [inline helpers/polyfills.test.ts]: is there a reason why you're requiring zone.js here and below as `'zone.js/dist/zone-node'`?

**Comment by gdi2290** [comment]
LGTM other than the comments

File: helpers/polyfills.test.ts (hunk 1 above)
Comment (devCrossNet) [inline helpers/polyfills.test.ts]: sorry for that, it was a copy/paste error

**Comment by devCrossNet** [comment]
@gdi2290 fixed

**Comment by devCrossNet** [comment]
@gdi2290 this PR is not WIP anymore. Do you have something else that I should change?

**Comment by gdi2290** [comment]
will be merged after AoT

**Comment by devCrossNet** [comment]
@gdi2290 I rebased and added the old build-step just for the test-task. I hope that is ok for you.

**Comment by gdi2290** [comment]
@googlebot I agree

**Comment by gdi2290** [comment]
@googlebot @google-admin @googleassistant I confirm

**Pull request merged**